src/intertypes/julia.jl

```diff
@@ -307,7 +307,7 @@ end
 
 function include_intertypes(into::Module, file::String, imports::AbstractVector)
   endswith(file, ".it") || error("expected a file ending in \".it\"")
-  name = Symbol(chop(file; tail=3))
+  name = Symbol(basename(chop(file; tail=3)))
```

Comment:
@olynch, any unforeseen consequence of this with respect to hierarchically named intertypes modules? We don't actually want path separators in these names right?